merlin/study/batch.py

```diff
@@ -335,12 +335,16 @@ def construct_worker_launch_command(parsed_batch: Dict, nodes: int) -> str:
     scheduler_legend: Dict = construct_scheduler_legend(parsed_batch, nodes)
     workload_manager: str = get_batch_type(scheduler_legend)
 
+    print(f"parsed_batch: {parsed_batch}")
```

Comment:
Do you want to leave in these print statements? Maybe they should have logger if clauses?

Reply:
whoops I didn't mean to leave this in there, good catch